### PR TITLE
Add debugging night vision glasses for Reyblood

### PR DESCRIPTION
**Pull request by Markolie**
*This pull request should be merged after https://github.com/ParadiseSS13/Paradise/pull/11357 and is still pending head admin approval.*

**What does this PR do:**
One of our players, Reyblood, is unable to play the game with the new lighting system. They are using a laptop from 2002, and rely on the integrated graphics of their CPU to play the game. The new lighting system is too heavy on said CPU. 

As such, pending headmin approval, this pull request adds a pair of NODROP debugging night vision glasses to the game that completely remove lighting (like X-Ray does, without the actual X-Ray). They can also be used by coders for debugging.

I realize that this may not be entirely balanced, but I think this player being able to play the game is more important. They have volunteered for a blanket antag ban, but I personally do not believe that is necessary.

This is what they see with the new system: https://prnt.sc/nkvpka

### CODE REVIEW

**Comment by Kyep** [comment]
I approve of the concept of goggles that negate lighting when worn.
Aside from them being useful for players like reyblood on ancient devices, they might also be useful for coders/maints, to find out if a certain issue is caused by lighting or not.

The PR needs to be modified, though, so that these are listed as a debugging item, not a fluff item, and they're usable by other ckeys. That way we can spawn them in for coders/whatnot during testing.

**Comment by Markolie** [comment]
Sure, I'll modify it now.

File: code/modules/clothing/glasses/glasses.dm [hunk 1]
```diff
@@ -177,6 +177,15 @@
 	flags = NODROP
 	flags_cover = null
 
+/obj/item/clothing/glasses/material/lighting
+	name = "Neutron Goggles"
```
Comment (Fox-McCloud) [inline code/modules/clothing/glasses/glasses.dm]: wont' this mean they they can see objects through walls?

Comment (Markolie) [reply code/modules/clothing/glasses/glasses.dm]: Negative, as the sight_flags are reset on the subtype.

**Comment by taukausanake** [comment]
> completely remove lighting (like X-Ray does, without the actual X-Ray)

So this makes everything you see visible in perfect lighting but it respects line-of-sight, correct? I'm just trying to make sense of it myself because it sounds really neat

**Comment by Markolie** [comment]
Aye, so you can’t actually see more than two tiles away in darkness anyway (unless you’re playing a species with higher darksight). In practise it’s very ugly, as it’s effectively like the old squared lighting from many years back.

**Comment by Ralta** [comment]
For any other game, there comes a point where you say "Look, you'll just have to upgrade."

That a solution like this can be built for specific players is, to me, really telling of how the SS13 community works. Good job, @markolie

**Comment by taukausanake** [comment]
If I could, I would run SS13 on Windows XP and _**only Windows XP**_. It's just that the BYOND client doesn't have its old look anymore which clashes with the ancient look for me. Also you need a minimum IE version and I can't remember if XP has that.

_I wonder if Windows Embedded would work..._

**Comment by Markolie** [comment]
Reyblood asked for a status update - is this good to be merged? They can't play the game properly until it's merged and added to the fluff item list (although we can manually set the lighting_alpha on their eyes in the meantime).

**Comment by datlo** [comment]
Is there a way this could be made opt-in for people who wants it as a loadout item or something? Lighting has a noticeable performance effect on my laptop and while its not as bad and still technically playable for me any sort of fighting situation is just impossible to play properly in now.

**Pull request merged**